parser/parse.go

```diff
@@ -458,6 +458,7 @@ func (b *Builder) findTypesIn(pkgPath importPathString, u *types.Universe) error
 	for _, f := range b.parsed[pkgPath] {
 		if strings.HasSuffix(f.name, "/doc.go") {
 			tp := u.Package(string(pkgPath))
+			tp.Comments = []string{}
```

Comment:
if tp.Comments == nil, it is still valid to append to it.  This initialization line is effectively a no-op, unless there were contents that you wanted to throw away?